### PR TITLE
8264539: Improve failure message of java/nio/file/WatchService/SensitivityModifier.java

### PR DESCRIPTION
**Pull request by bplb**
Please consider this request to improve the message printed when no `ENTRY_MODIFY` events are received. In the polling implementation of `WatchService` currently used on macOS, it would be useful to know the sensitivity of polling in effect for the failing case. It might indicate, for example, that low sensitivity should not be used in the test.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8264539](https://bugs.openjdk.java.net/browse/JDK-8264539): Improve failure message of java/nio/file/WatchService/SensitivityModifier.java

### Reviewers
 * [Alan Bateman](https://openjdk.java.net/census#alanb) (@AlanBateman - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/3290/head:pull/3290` \
`$ git checkout pull/3290`

Update a local copy of the PR: \
`$ git checkout pull/3290` \
`$ git pull https://git.openjdk.java.net/jdk pull/3290/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 3290`

View PR using the GUI difftool: \
`$ git pr show -t 3290`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/3290.diff">https://git.openjdk.java.net/jdk/pull/3290.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/nio/file/WatchService/SensitivityModifier.java [hunk 1]
```diff
@@ -108,7 +120,10 @@ static void doTest(Path top) throws Exception {
                 if (eventReceived) {
                     System.out.println("Event OK");
                 } else {
-                    throw new RuntimeException("No ENTRY_MODIFY event received for " + file);
+                    Path parent = file.getParent();
+                    String msg = String.format("No ENTRY_MODIFY event received for %s (dir: %s, sensitivity: %d)",
+                            file, parent, pathToTime.get(parent));
+                    throw new RuntimeException(msg);
```
Comment (AlanBateman) [inline test/jdk/java/nio/file/WatchService/SensitivityModifier.java:126]: Not clear to me that this helps. Have you looked at changing the 1s at L104. I assume the issue is that the test needs the idle (no events) time increased to deal with cases where the watcher is slow to queue events.

Comment (bplb) [reply test/jdk/java/nio/file/WatchService/SensitivityModifier.java:126]: That's a better idea but I think this change should go in also. I have not in any case been able to reproduce it. There is no record of on which specific machines it was sighted.

Comment (AlanBateman) [reply test/jdk/java/nio/file/WatchService/SensitivityModifier.java:126]: Okay.

**Review by AlanBateman — APPROVED** [review]
Changes look okay although I don't think changing POLL_TIMEOUT_SECONDS vary by platform is necessary.

**Pull request closed without merging**